Proctor_Brad/Algorithms/algo100516.html

```diff
@@ -0,0 +1,40 @@
+<html>
+<head>
+  <title></title>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+  <script type="text/javascript">
+  /*
+  $(docuemnt).ready(function(){
+    // ... your assignment here
+  });
+  */
+
+  $(function(){
+    arr = str.split(" ");
+    return arr.join("");
+  });
+  $(function(){
+    arr = str.split(" ");
+    for(var i = 0; i < arr.length(); i++){
+        arr[i] = a[i][0];
+    }
+    arr.join("").toUpperCase();
+    return arr;
+  });
+  $(function(){
+    arr = str.split("");
+    arr2 = [];
+    for(var i = 0; i < arr.length(); i++){
+        if(arr[i] >= 0 && arr[i] <= 9){
+            arr2.push(arr[i]);
+        }
+    }
+    arr2.join("");
+    return arr2;
+  });
```

Comment:
where is str defined? you need to pass it in as a parameter to these functions. also, don't forget to test the functions out by calling them with test/sample data and logging the result to console.log